addons/circulation/CfgFunctions.hpp

```diff
@@ -8,6 +8,12 @@ class CfgFunctions {
             class cprSuccess {
                 file = QPATHTOF(functions\fnc_cprSuccess.sqf);
             };
+            class CheckBloodPressure {
```

Comment:
instead of doing such overwrites, change callbackSuccess of those actions and point it to our functions